base/abstractarray.jl

```diff
@@ -374,6 +393,8 @@ first(a::AbstractArray) = a[first(eachindex(a))]
 Get the first element of an iterable collection. Return the start point of an
 [`AbstractRange`](@ref) even if it is empty.
 
+See also: [`only`](@ref), [`firstindex`](@ref), [`last`](@ref), [`tail`](@ref Base.tail).
```

Comment:
```suggestion
See also: [`only`](@ref), [`firstindex`](@ref), [`last`](@ref).
```

I wouldn't generally consider tail to be similar. We could mention `popfirst!`, but I'm also not sure that is helpful.

Reply:
Not so much similar, as often used together, was my thinking -- things like `g(x) = (f(first(x)), map(g, tail(x))...)`. But you may not know whether it's called `rest` or `allbutfirst` or something.

Reply:
`popfirst!` would be more typical for that need (or perhaps `Base.rest`)